package.json

```diff
@@ -44,7 +44,7 @@
     "chai": "^4.2.0",
     "mocha": "^6.2.0",
     "nyc": "^14.1.1",
-    "semantic-release": "8.2.0",
```

Comment:
am n ot sure if it'll help , there was some issue with 15.x in CI , chek this https://github.com/fabric8-analytics/fabric8-analytics-lsp-server/pull/85

Reply:
Thanks @invincibleJai . I had updated the build scripts to work with latest semantic-release module. PTAL.